README.md

```diff
@@ -80,27 +80,70 @@ make docker-dev
 
 After either command, you will find the `mediator` API running at http://127.0.0.1:8000 and the `guardian` API at http://127.0.0.1:8001
 
-## 🐍 Developing with Python
+## 🐍 3. Developing with Python
+
+Developing with Python provides the fastest developer inner loop (speed from code changes to seeing effects of changes), but is more work to set up initially. It works on Mac and Linux. It also works via [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/about) on Windows.
+
+### ✅ 3.1. Windows Prerequisites
+
+On Windows you can use an IDE of your choice in Windows, and run the make and Python commands in WSL which will expose API's in Windows. However, developing with Python on Windows involves additional setup that is not required for Linux or Mac.
+
+1. Install [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install)
+2. Install [Ubuntu](https://www.microsoft.com/en-us/p/ubuntu/9nblggh4msv6?ocid=9nblggh4msv6_ORSEARCH_Bing&rtc=1&activetab=pivot:overviewtab) (other Linux distributions should also work with minor modifications to the instructions below)
+3. Install [pyenv prerequisites](https://github.com/pyenv/pyenv/wiki#suggested-build-environment). Technically you could just install Python and it would be simpler, but this approach will provide more flexibility. To install the prerequisites:
+
+```bash
+sudo apt-get update
+sudo apt-get install make build-essential libssl-dev zlib1g-dev \
+  libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+  libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+```
+
+4. Install pyenv via [pyenv-installer](https://github.com/pyenv/pyenv-installer) and add it the startup scripts:
+
+```bash
+curl https://pyenv.run | bash
+echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
+echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
+echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n eval "$(pyenv init -)"\nfi' >> ~/.bashrc
+sed -Ei -e '/^([^#]|$)/ {a \
+  export PYENV_ROOT="$HOME/.pyenv"
+  a \
+  export PATH="$PYENV_ROOT/bin:$PATH"
+  a \
+  ' -e ':a' -e '$!{n;ba};}' ~/.profile
+echo 'eval "$(pyenv init --path)"' >>~/.profile
+
+echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+```
+
+5. Restart shell
+6. Install Python 3.9 via pyenv
+
+```bash
+pyenv install 3.9.9
+pyenv global 3.9.9
+```
 
-Developing with Python provides the fastest developer inner loop (speed from code changes to seeing effects of changes), but is more work to set up initially.
+### ✅ 3.2 Mac/Linux Prerequisites
 
-### Quick Start
+Install [Python 3.9](https://www.python.org/downloads/). We additionally recommend [pyenv](https://github.com/pyenv/pyenv) to assist with Python version management (see detailed instructions in the Windows section above starting with Step 3).
```

Comment:
Installing pyenv was super complicated in WSL.  I don't technically know that it's as complicated on Mac (maybe Homebrew makes it easy?), but I'd imagine those instructions will work on Linux, thus the reference to "step 3".  Game to mention Homebrew or something, but I'd need confirmation from a Mac user.